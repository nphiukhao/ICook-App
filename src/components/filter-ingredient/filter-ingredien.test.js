import React from 'react'
import ReactDOM from 'react-dom';
import FilterIngredient from './filter-ingredient'


it(`Renders without crashing`, () => {
    const div = document.createElement('div')
    ReactDOM.render(<FilterIngredient />, div)
    ReactDOM.unmountComponentAtNode(div);
});