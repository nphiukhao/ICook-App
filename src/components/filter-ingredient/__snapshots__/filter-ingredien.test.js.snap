// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterIngredient component FilterIngredient snapshot test 1`] = `<FilterIngredient />`;
